app/tests/database.py

```diff
@@ -0,0 +1,140 @@
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import CollectionLink
+from stac_api.models import Item
+from stac_api.models import ItemLink
+from stac_api.models import Keyword
+from stac_api.models import Provider
+from stac_api.models import get_default_stac_extensions
+
+
+def create_collection():
+    '''Create a dummy collection db object for testing
+    '''
+    collection = Collection.objects.create(
+        crs=["http://www.opengis.net/def/crs/OGC/1.3/CRS84"],
+        created='2020-10-28T13:05:10.473602Z',
+        updated='2020-10-28T13:05:10.473602Z',
+        description='This is a description',
+        start_date=None,
+        end_date=None,
+        extent=[200000, 100000, 200001, 100005],
```

Comment:
Not compatible with the latest collection model any more.

Reply:
Well it depends on who is merging first :-). Each PR must be compatible with the develop branch and not on other PR. So Either I merged it now (this changes needs to be part of your PR then) or I wait that you merge the Collection serializer and do the changes within this PR :-)